### PR TITLE
Address comments

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: test/integration/etcd_storage_path_test.go [hunk 1]
```diff
@@ -890,8 +890,12 @@ func testEtcdStoragePath(t *testing.T, etcdServer *etcdtest.EtcdTestServer, gett
 
 		mapping, err := mapper.RESTMapping(gvk.GroupKind(), gvk.Version)
 		if err != nil {
-			kindSeen.Insert(kind)
-			if kindWhiteList.Has(kind) {
+			if kindWhiteList.Has(gvk.String()) {
```
Comment (sttts) [inline test/integration/etcd_storage_path_test.go]: @enj I readded @ncdc's "hack"

File: pkg/cmd/server/kubernetes/master/master_config.go [hunk 2]
```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```
Comment (sttts) [inline pkg/cmd/server/kubernetes/master/master_config.go]: @ncdc With this we have RBAC issues (seen in etcd storage test):
```
--- FAIL: TestEtcd2StoragePath (55.67s)
	etcd_storage_path_test.go:960: failed to create stub for Role from github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/apis/rbac/v1beta1: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:""}, Status:"Failure", Message:"roles.rbac.authorization.k8s.io \"r1b1\" is forbidden: attempt to grant extra privileges: [{[create] [authorization.k8s.io] [selfsubjectaccessreviews] [] []}] user=&{system:admin  [system:cluster-admins system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[]", Reason:"Forbidden", Details:(*v1.StatusDetails)(0xc432fffea0), Code:403}}
	etcd_storage_path_test.go:960: failed to create stub for ClusterRole from github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:""}, Status:"Failure", Message:"clusterroles.rbac.authorization.k8s.io \"cr1a1\" is forbidden: attempt to grant extra privileges: [{[create] [authorization.k8s.io] [selfsubjectaccessreviews] [] []}] user=&{system:admin  [system:cluster-admins system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[]", Reason:"Forbidden", Details:(*v1.StatusDetails)(0xc42ddef040), Code:403}}
	etcd_storage_path_test.go:960: failed to create stub for Role from github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/apis/rbac/v1alpha1: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:""}, Status:"Failure", Message:"roles.rbac.authorization.k8s.io \"r1a1\" is forbidden: attempt to grant extra privileges: [{[create] [authorization.k8s.io] [selfsubjectaccessreviews] [] []}] user=&{system:admin  [system:cluster-admins system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[clusterroles.rbac.authorization.k8s.io \"cr1b1\" not found]", Reason:"Forbidden", Details:(*v1.StatusDetails)(0xc4395840a0), Code:403}}
	etcd_storage_path_test.go:960: failed to create stub for ClusterRole from github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/apis/rbac/v1beta1: &errors.StatusError{ErrStatus:v1.Status{TypeMeta:v1.TypeMeta{Kind:"", APIVersion:""}, ListMeta:v1.ListMeta{SelfLink:"", ResourceVersion:""}, Status:"Failure", Message:"clusterroles.rbac.authorization.k8s.io \"cr1b1\" is forbidden: attempt to grant extra privileges: [{[create] [authorization.k8s.io] [selfsubjectaccessreviews] [] []}] user=&{system:admin  [system:cluster-admins system:authenticated] map[]} ownerrules=[] ruleResolutionErrors=[[clusterroles.rbac.authorization.k8s.io \"cr1a1\" not found, clusterroles.rbac.authorization.k8s.io \"cr1b1\" not found]]", Reason:"Forbidden", Details:(*v1.StatusDetails)(0xc4321c38b0), Code:403}}
```

Comment (ncdc) [reply pkg/cmd/server/kubernetes/master/master_config.go]: You'll either have to enable the upstream hook, or disable testing the kube rbac kinds. Or disable the rbac authorizer, maybe. Are there other options?

Comment (sttts) [reply pkg/cmd/server/kubernetes/master/master_config.go]: This comes from the rbac registry validation. I.e. the upstream authorizer is not used. The validation complains in `ConfirmNoEscalation`. We can
- tweak the storage test to write more innocent values (if that is possible)
- ignore them completely in the storage test (and loosing test coverage)
- leave the bootstrap hook enabled.

@liggitt @enj what do you prefer? Patches for the rebase branch welcome.

Comment (enj) [reply pkg/cmd/server/kubernetes/master/master_config.go]: https://github.com/kubernetes/kubernetes/pull/44633 has correct values for the RBAC resources.

Comment (liggitt) [reply pkg/cmd/server/kubernetes/master/master_config.go]: for RBAC, can run the test with a user in the `system:masters` group, which the RBAC registry will allow to escalate

Comment (sttts) [reply pkg/cmd/server/kubernetes/master/master_config.go]: I would vote for turning this into a direct post-rebase follow-up as it is only cosmetic. The storage test runs fine now, but we have the bootstrap hook with its /healthz endpoint.

@liggitt are you ok with a follow-up?

**Pull request merged**